content/docs/quickstart.mdx

```diff
@@ -57,39 +84,72 @@ Add the configuration below to `docker-compose.yaml`:
 docker compose up
 ```
 
-Access the **verify route** you built in your policy: `https://verify.localhost.pomerium.io`
+## Access the protected web app
 
-If you get a self-signed certificate warning, see [Handle Self-Signed Certificate Warning](/docs/internals/troubleshooting#handle-self-signed-certificate-warning) to bypass it.
+Go to the [Verify URL](https://verify.localhost.pomerium.io) you defined in `config.yaml`.
 
-You should be redirected to the **verify** service. You'll see a page like this:
+### Self-signed certificate warning
 
-![Verify page](./img/quickstart/verify-service.png)
+Because you don't have a valid certificate, Pomerium generates a self-signed one for you. This will prompt your browser to throw a self-signed certificate warning.
 
-Although identity verification failed, you successfully integrated Pomerium with the upstream verify service.
+To bypass the warning:
 
-:::tip
+<Tabs>
+<TabItem value="Chrome" label="Chrome">
 
-Because this guide doesn't include a [signing key](/docs/reference/signing-key) in the configuration, identity verification will fail.
+1. Select **Advanced**
+1. Select **Proceed to verify.localhost.pomerium.io (unsafe)**
 
-See [Identity Verification](/docs/capabilities/getting-users-identity) for more information on how Pomerium can use JWTs for authentication.
+If you don't see an **Advanced** option:
 
-:::
+1. Click anywhere in the window
+1. Type "thisisunsafe" (no spaces)
+1. Make sure **Reload** is selected
+1. Select **Enter**
 
-## Next Steps
+</TabItem>
+<TabItem value="Safari" label="Safari">
+
+1. Select **Show Details**
+1. Select **visit this website**
+1. In the confirmation popup, select **Visit Website**
+
+</TabItem>
+<TabItem value="FireFox" label="FireFox">
+
+1. Select **Advanced**
+1. Select **Accept the Risk and Continue**
+
+</TabItem>
+</Tabs>
+
+Pomerium will redirect you to our hosted identity provider to authenticate.
 
-If you want to [try Enterprise](https://www.pomerium.com/enterprise-sales/), check out the [Enterprise with Docker quickstart](/docs/deploy/enterprise/quickstart).
+Then, it will redirect you to the **Verify** service. You'll see a page like this:
 
-If you want to try connecting Pomerium with other services, see some of our [Guides](/docs/guides).
+![Accessing the Verify web app behind Pomerium](./img/quickstart/identity-verified.png)
```

Comment:
Unfortunately this won't work with the latest version of the Verify app (since https://github.com/pomerium/verify/commit/b8ce8e42f5131d523270e3efbfa1ef7ec5ef3a1c). The Verify app will now correctly show a TLS error because Pomerium does not have a trusted certificate.